docs/api-status.md

```diff
@@ -56,6 +56,11 @@ Snapshot.Set | v0.10.0 |  |
 
 Name | Added in Version | Expected Stable Version | 
 ---- | ---------------- | ----------------------- | 
+API.ListUsersBuckets | v0.13.0 | v0.14.0 | 
```

Comment:
Should this be Added in 0.14 and Expected stable in 0.15? @phlogistonjohn

Reply:
Thanks for noticing that. We're aware of the workflow issues with the preview policy and "stale" versions numbers in docs/json.
The "Added in Version" should always be the upcoming go-ceph release, so in this case "v0.15.0". The script automatically puts Y+2 for stable, so "v0.17.0" but that is negotiable.

Reply:
fixed